### PR TITLE
Add 0.146 release notes

### PR DESCRIPTION
**Pull request by martint**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.146.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+=============
+Release 0.146
+=============
+
+* Fix error in map_concat when the second map is empty.
+* Add support for ``VARCHAR(n)`` to Blackhole connector.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: We call it "Black Hole" elsewhere in the docs

File: presto-docs/src/main/sphinx/release/release-0.146.rst [hunk 2]
```diff
@@ -0,0 +1,9 @@
+=============
+Release 0.146
+=============
+
+* Fix error in map_concat when the second map is empty.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: Link to the function

File: presto-docs/src/main/sphinx/release/release-0.146.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+* Add support for ``VARCHAR(n)`` to Blackhole connector.
+* Add initial support for ``DECIMAL`` to Raptor connector.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: We don't document Raptor changes

File: presto-docs/src/main/sphinx/release/release-0.146.rst (hunk 1 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: Split this out into a separate section for "Black Hole Changes"

File: presto-docs/src/main/sphinx/release/release-0.146.rst (hunk 2 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: Add a "General Changes" section for this

File: presto-docs/src/main/sphinx/release/release-0.146.rst [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+* Add support for Cassandra 3.0.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: Add a "Cassandra Changes" section for this

File: presto-docs/src/main/sphinx/release/release-0.146.rst [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+* Add MongoDB connector.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: Make this a link
```
* Add :doc:`/connector/mongodb`.

File: presto-docs/src/main/sphinx/release/release-0.146.rst (hunk 1 above)
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: There's only one change, so it seems unnecessary to have a whole section for it.

File: presto-docs/src/main/sphinx/release/release-0.146.rst (hunk 4 above)
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: Same here, one change, so it doesn't seem worth it.

File: presto-docs/src/main/sphinx/release/release-0.146.rst (hunk 1 above)
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.146.rst]: It makes it easier to read. Plus, there might only be one change now, but there could be another release note added, then it has to move. Better to always follow the rule of separate sections per connector.

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**